### DISCUSSION TITLE
Elasticsearch

### DISCUSSION THREAD

**Discussion by antmar904** [LETHAL-FORENSICS/MemProcFS-Analyzer#7] [General]
Hi.

Once analysis is done and the analyzer has stopped and all processes are closed is there a way to load the data extracted from memory back into Elasticsearch or do I need to analyze the memory dump again to get back to Elastic?

**Comment by evild3ad** [comment]
Should work...the indexed JSON data is stored in "./Tools/Elasticsearch/data", but I haven't tested it yet.

You can comment some functions in the Main (bottom of the script) to speed up things or launch Elasticsearch and Kibana manually.

![Main](https://user-images.githubusercontent.com/8110531/121143749-ab740600-c83d-11eb-8900-52e057054d2d.png)

I think I will split all features to more functions in v.04,

**Reply by antmar904** [reply]
awesome, ill give it a try!  I also have my own instance of SOF-ELK running.  I'm going to try to import the JSON data from the analysis and into it's own index.

**Comment by antmar904** [comment]
What data is imported into Elastic, forensic\json ?
